plugins/sslutils.c

```diff
@@ -191,17 +191,6 @@ int np_net_ssl_read(void *buf, int num) {
 	return SSL_read(s, buf, num);
 }
 
-int np_net_ssl_check_cert(int days_till_exp_warn, int days_till_exp_crit){
```

Comment:
same for this change? Is it part of the gnulib update?

Reply:
This one is not actually required for the gnulib update, but it fixes a compiling problem for the gentoo folks where the compilers do actually complain if you try to use functions which the compiler does not (yet) know about.

Reply:
But for more clearness, I can separate this

Reply:
nope, it's fine. Just wanted to know